storage/rocksdb/ha_rocksdb.cc

```diff
@@ -10778,8 +10778,8 @@ bool can_use_bloom_filter(THD *thd,
     return can_use;
   }
 
-  rocksdb::Options opt = rdb->GetOptions(kd.get_cf());
-  if (opt.prefix_extractor)
+  auto prefix_extractor= kd.get_extractor();
```

Comment:
I'm pretty sure this will create a copy of the shared_ptr, which will inc/dec reference counts.